### ISSUE TITLE
Pass command line flags to Bromite

### ISSUE THREAD

**Issue by darylrobert** [bromite/bromite#532]
### Bromite version

Version: 81.0.4044.106
Arch: arm64
Android version: 9.0
Device model: ginkgo M1908C3JG

### Is this bug about the SystemWebView?
No

### Is the bug reproducible with latest version?

Yes, 81.0.4044.106

### Can the bug be reproduced with corresponding Chromium version?

No, 81.0.4044.106

### Is the bug a crash?

No

### Describe the bug
I'm trying to make a mini dump from a tab crash (Aw, Snap!)

Reproducing steps:
1. Open either [https://optout.aboutads.info/?c=3&lang=EN](https://optout.aboutads.info/?c=3&lang=EN), [https://optout.aboutads.info/?c=2&lang=EN](https://optout.aboutads.info/?c=2&lang=EN), or [https://optout.networkadvertising.org/?c=1](https://optout.networkadvertising.org/?c=1)
2. Wait page to complete load
3. Tap on "OPT OUT FROM ALL", or "select all" then "SUBMIT YOUR CHOICES"
4. Aw, Snap!

This issue is not reproducible with corresponding Chromium built by Bromite (org.bromite.chromium) version.

How I tried to set the command line flags to enable crashpad:
- adb shell echo "--enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache" > /data/local/chrome-command-line
- adb shell echo "--enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache" > /data/local/tmp/chrome-command-line and enabling #enable-command-line-on-non-rooted-device and restarting Bromite by force stopping it and launching it again from launcher
- tried  "_ --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache", "--_ --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache",  "chrome --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache",  "chromium --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache",  "bromite --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache", "chrome _ --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache",  "chromium _ --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache",  "bromite _ --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache",  "chrome --_ --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache",  "chromium --_ --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache",  "bromite --_ --enable-crash-reporter --crash-data-dir=/data/data/org.bromite.bromite/cache" 
basically any flag permutation I found on the Internet
- chmod 755 chrome-command-line and chmod 777 chrome-command-line

for some reason this person is able to pass command lines https://github.com/wchen342/ungoogled-chromium-android/issues/14

This issue is reproducable with corresponding Chromium built by Bromite (org.bromite.chromium) version.

**Comment by csagan5** [comment]
@darylrobert try to be less colorful in issue titles

For the crash, you can submit the `adb logcat` section with the crash details. There should be tombstones too.

For the issue of not being able to pass command line flags, I will investigate.

**Comment by darylrobert** [comment]
It's one of the most frustrating things and all the documentation for Chromium for this supposed feature, neither does work.

**Comment by csagan5** [comment]
> neither does work.

What does not work?

**Comment by darylrobert** [comment]
basically any documentation that supposedly explain how to pass parameters to Chromium Android.
I'm sorry for my behavior.

anyways, the crash didn't showed up in logcat, but it does spit a tombstone:
[tombstone_01.txt](https://github.com/bromite/bromite/files/4510285/tombstone_01.txt)
Sidenote: I need to patch the McDonald's app using a certain apk modding tool to allow it to run on a rooted device.

**Comment by csagan5** [comment]
> basically any documentation that supposedly explain how to pass parameters to Chromium Android.

I do not write that documentation.

It worked for me at first try: https://stackoverflow.com/a/39431577

It is the same explanation provided in [official documentation](https://www.chromium.org/developers/how-tos/run-chromium-with-flags#TOC-Android):
> For production build on a non-rooted device, you need to enable "Enable command line on non-rooted devices" in chrome://flags, then set command line in /data/local/tmp/chrome-command-line. When doing that, mind that the first command line item should be a "_" (underscore) followed by the ones you actually need. Finally, manually restart Chrome ("Relaunch" from chrome://flags page might no be enough to trigger reading this file). See https://crbug.com/784947.

The upload of crash reports is disabled in Bromite, but the creation of tombstones is not.

The crash you found is not in Bromite but in `om.dp.logcatapp`

**Comment by darylrobert** [comment]
Huh, weird. I'll try to make Bromite crash again.
